src/pyedb/dotnet/edb_core/stackup.py

```diff
@@ -2130,7 +2134,7 @@ def _import_xml(self, file_path, rename=False):
                 layer = {"name": l.attrib["Name"]}
                 for k, v in l.attrib.items():
                     if k == "Color":
-                        layer[k.lower()] = [int(x * 255) for x in list(matplotlib.colors.to_rgb(v))]
+                        layer[k.lower()] = [int(x * 255) for x in list(colors.to_rgb(v))]
```

Comment:
If `colors` is equal to `None`, wouldn't that LOC raise an exception ?

Reply:
Yes, but here I am following what was done in this module for numpy and pandas as well.  These cannot be imported in IronPython and are not in the base dependency set so using functions that depend on them will fail unless you install pyedb[full].

Open to suggestions on how to better handle this, but this is how it is being done for the other two moduels currently.

Reply:
Ah, I see the check for np is None before it is used, I'll add this.

Reply:
@SMoraisAnsys I've added a check for colors being None to gracefully fail in this case, in the same way as is done if np is None elsewhere.

Reply:
Yeah, that will avoid the problem mentioned. Thanks for handling it :)